collector/compile-benchmarks/include-blob/src/main.rs

```diff
@@ -0,0 +1,10 @@
+const BLOB_BINARY: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/blob-binary"));
+const BLOB_STRING: &str = include_str!(concat!(env!("OUT_DIR"), "/blob-string"));
+
+fn main() {
+    println!(
+        "Binary blob last element: {}",
+        BLOB_BINARY[BLOB_BINARY.len() - 1]
+    );
+    println!("String blob: {BLOB_STRING}");
```

Comment:
The asymmetry in the printing here feels odd. Maybe just print both blobs with `{:?}`, so they're basically the same?

Reply:
Fair enough! Done.